apps/web/src/screens/Setting/index.tsx

```diff
@@ -1,12 +1,15 @@
-import { Stack } from '@walless/ui';
+import { Stack, Text } from '@walless/ui';
 import { appState } from 'state/app';
 import { useSnapshot } from 'utils/hooks';
 
-import Delimiter from './components/Delimiter';
-import FollowUs from './components/FollowUs';
-import Header from './components/Header';
-import HelpCenter from './components/HelpCenter';
-import MyWallets from './components/MyWallets';
+import {
+	Delimiter,
+	FollowUs,
+	Header,
+	HelpCenter,
+	LogOut,
+	MyWallets,
+} from './components';
```

Comment:
prefer `import from './components/ComponentName';` than this, re-export using `index.ts` file may cause unexpected circular deps.